File: NetFramework/DashboardLayout/VB/DashboardLayoutDemo/DemoControls/TrueDBGrid.vb
Imports C1.Win.TrueDBGrid

Namespace DashboardDemo.Controls
    Public Class DemoTrueDBGrid
        Inherits C1TrueDBGrid
        Implements IDemo

        Public Sub Init() Implements IDemo.Init
            DataSource = SampleHelper.Instance.GetSourceTable("Customers", Resources.Customers)
        End Sub
    End Class
End Namespace

File: NetFramework/DashboardLayout/CS/DashboardLayoutDemo/DemoControls/TrueDBGrid.cs
using C1.Win.C1TrueDBGrid;

namespace DashboardDemo.Controls
{
    public class DemoTrueDBGrid : C1TrueDBGrid, IDemo
    {
        public void Init()
        {
            DataSource = SampleHelper.Instance.GetSourceTable("Customers", Properties.Resources.Customers);
        }
    }
}
